src/test/java/racingcar/domain/CarNamesTest.java

```diff
@@ -18,12 +20,12 @@ void of(String carNames, int expectCarCount) {
         assertThat(names.carCount()).isEqualTo(expectCarCount);
     }
     
-    @DisplayName("자동차 이름 길이가 유효하지 않다면 IllegalArgumentException 예외를 발생시킨다.")
+    @DisplayName("자동차 이름 길이가 유효하지 않다면 InvalidCarNameException 예외를 발생시킨다.")
     @ParameterizedTest
     @ValueSource(strings = { "", "pobi, honux,invalid"})
     void of_invalid_car_name(String invalidCarNames) {
         assertThatThrownBy(() -> {
             CarNames.of(invalidCarNames);
-        }).isInstanceOf(IllegalArgumentException.class).hasMessageMatching("The car name is at least 1 characters and up to 5 characters.");
+        }).isInstanceOf(InvalidCarNameException.class).hasMessageMatching("The car name is at least 1 characters and up to 5 characters.");
```

Comment:
개행을 해주시면 가독성에 도움이 될 것 같아요! 😃 
```suggestion
        }).isInstanceOf(InvalidCarNameException.class)
          .hasMessageMatching("The car name is at least 1 characters and up to 5 characters.");
```